test/Mono.Linker.Tests.Cases/CommandLine/Dependencies/CustomStepUser.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using Mono.Linker;
+using Mono.Linker.Steps;
+
+namespace CustomStep
+{
+	public class CustomStepUser : IStep
+	{
+		public void Process(LinkContext context)
+		{
+			if (context.TryGetCustomData ("NewKey", out var value))
+				context.LogMessage ("Custom step added with custom data of " + value);
```

Comment:
This should throw if the value is not the expected `UserValue` - printing it out doesn't actually verify the value in any way.

Reply:
The value is verified at https://github.com/mono/linker/pull/1128/files#diff-26f773e2a62fedeb70de25da1d59505cR15. Do you think that's not enough?

Reply:
Sorry - I missed that part - looks good.